sdk/tseal/tSeal.cpp

```diff
@@ -217,8 +217,8 @@ extern "C" sgx_status_t sgx_unseal_data(const sgx_sealed_data_t *p_sealed_data,
     }
 
     if((p_additional_MACtext_length != NULL) && 
```

Comment:
The check for `p_decrpted_text_length` also needs to be updated, right?
Line 205:

> if (!sgx_is_within_enclave(p_decrypted_text_length, sizeof(p_decrypted_text_length)))
     {
              return SGX_ERROR_INVALID_PARAMETER;
     }

Reply:
Yeah. You're right. Updating

Reply:
fixed in b38e0d8cdd4dee8b8bfab65bd06b3b39f7886c77

Thanks!